src/Components/Filters/Filters.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import './Filters.css';
+
+class Filters extends Component {
+
+  showAll = () => {
+    this.props.fromParentShowNotSortedListOfTasks();
+  }
+
+  filterByCompletedAndActiveTasks = (isCompleted) => {
+    let newTasks = [];
+    for (let value of this.props.listOfTasks) {
+      if (value.isCompleted === isCompleted) {
+        newTasks.push(value);
+      }
+    }
+    this.props.fromParentAddSortedListOfTasks(newTasks);
+  }
+
+  filterByTitle = () => {  
+    this.props.fromParentAddSortedListOfTasks(this.props.listOfTasks.sort((a, b) => a.title.localeCompare(b.title)));
+  }
+
+  filterByOrigin = () => {  
+    this.props.fromParentAddSortedListOfTasks(this.props.listOfTasks.sort((a, b) =>a.data - b.data));
+  }
+
+  render() {
+    return (
+      <div className="Filters">
+        <div className = "filter-block" id="filter1" onClick={this.showAll}>SHOW ALL</div>
+        <div className = "filter-block" id="filter2" onClick={()=>this.filterByCompletedAndActiveTasks(false)}>SHOW ACTIVE</div>
```

Comment:
Request!
In case you have more than one attribute inside HTML tag, it should be formatted to the next example:

```
<div className="filters">
    <div 
        className="filters__block block block--show-all" 
        onClick={this.showAll}
    >
        SHOW ALL
    </div>
    <div 
        className="filters__block block block--show-active" 
        onClick={() => this.filterByCompletedAndActiveTasks(false)}
    >
        SHOW ACTIVE
    </div>
    <div 
        className="filters__block block block--show-completed" 
        onClick={() => this.filterByCompletedAndActiveTasks(true)}
    >
        SHOW COMPLETED
    </div>
    <div 
        className="filters__block block block--sort-by-alphabet" 
        onClick={this.filterByTitle}
    >
        SORT BY A-Z
    </div>
    <div 
        className="filters__block block block--sort-by-origin" 
        onClick={this.filterByOrigin}
    >
        SORT BY ORIGIN
    </div>
</div>
```